frontend/13f-app/src/components/companydata.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+import axios from 'axios'
+
+function Company () {
+  const [company, setCompany] = useState()
+  const location = useLocation()
+  const param = new URLSearchParams(location.search)
+  const name = param.get('name')
+
+  useEffect(() => {
+    axios.get(`http://localhost:5000/company/${name}`)
+      .then(res => {
+        const companydata = res.data
+        setCompany(companydata)
+      })
+  }, [])
+  return (
+        <div>
+            <h1>{companydata.company_name}</h1>
```

Comment:
```suggestion
        const companyData = res.data
        setCompany(companyData)
      })
  }, [])
  return (
        <div>
            <h1>{companyData.company_name}</h1>
```
Use camel case to effectively name variables